SmartSaver.MVC/Views/Authentication/Register.cshtml

```diff
@@ -7,47 +7,64 @@
     <link rel="stylesheet" type="text/css" href="~/css/generalDesign.css" />
     <title>@ViewBag.Title</title>
 </head>
+<br>
+<br>
+<body onload="UsernameCheck(); EmailCheck()" style="background-size:cover; background-image: url(https://images.unsplash.com/photo-1475650522725-015d35677789?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1050&q=80)">
+    <div class="card center" style="max-width: 500px; padding: 20px 20px 20px 20px;">
+        <div class="card-block">
+            <div class="col-md-12">
+                <form method="post">
```

Comment:
`asp-action`, `asp-controller` and `asp-route-returnUrl` is missing in the `<form>`. But this is my bad. Will change that